feature/experimental/gribi/ate_tests/dut_daemon_failure/dut_daemon_failure_test.go

```diff
@@ -299,16 +299,16 @@ func TestDUTDaemonFailure(t *testing.T) {
 		// Set parameters for gRIBI client clientA.
 		// Set Persistence to true.
 		clientA := &gribi.Client{
-			DUT:                  dut,
-			FibACK:               false,
-			Persistence:          true,
-			InitialElectionIDLow: 10,
+			DUT:         dut,
+			FibACK:      false,
```

Comment:
(nit) Should this be `FIBACK` given that we usually capitalise acronyms like this in Go?

Reply:
Done!